python_binding/src/stream_manager.rs

```diff
@@ -30,6 +30,9 @@ cfg_if! {
 /// manager=pravega_client.StreamManager("127.0.0.1:9090")
 /// // this manager can be used to create scopes, streams, writers and readers against Pravega.
 /// manager.create_scope("scope")
+///
+/// // optionally enable tls support using tls:// scheme
```

Comment:
Let's change the non-tls example to use the 'tcp' prefix so that it's consistent.